### ISSUE TITLE
Suggestion: Rewrite to Markdown or use the Wiki function

### ISSUE THREAD

**Issue by han44-illinois** [engrit-illinois/misc-handy-powershell-examples#1]
Given the purpose of this particular repository is to be read and not to actually run any of the code in it directly, I think it might make sense to change the structure a bit and put everything into a structure of .md files for better readability. Markdown supports inline code and code blocks.

`inline code`

```
blocks
of
code
```

It would also be useful to provide some basic formatting, such as headers and bulleted lists

## Headers
* List Item 1
* List item 2

Alternatively, it might make sense to utilize the Wiki, though I have never explored that side of github before.

Potential downside to rewriting in Markdown is that it makes the repo less useful when cloned down unless the reader has an application to preview Markdown.

Thought I'd create the issue first to present the idea and get your thoughts on it before starting any work involved, to see what you think.

**Comment by mmseng** [comment]
I agree for sure. It's primarily just a single PS1 file because that's how it started out when it was only a few examples. It's definitely grown to the point where that's a less readable, less maintainable, and more difficult to search format.

An MD file with headers for each example and inline/code blocks (using powershell syntax highlighting), and perhaps organized into themed sections, would be a much preferable format. If you care to do any of that, please feel free. If so, I'd recommend simply leaving the PS1 in place and copying over and annotating the examples piecemeal, as you have time. Or just copying the PS1 and reformatting it as you have time.

I'd be happy to contribute as well. Up to you.

**Comment by mmseng** [comment]
P.S. I added you as a maintainer.
